packages/migrate/src/commands/DbExecute.ts

```diff
@@ -36,21 +40,27 @@ ${chalk.bold('Usage')}
 ${chalk.bold('Options')}
 
 -h, --help   Display this help message
+
+${chalk.italic('Datasource input, only 1 must be provided:')}
      --url   URL of the datasource to run the command on
   --schema   Path to your Prisma schema file to take the datasource URL from
-   --stdin   Use the terminal standard input as the script to be executed.
-    --file   Path to a file. The content will be sent as the script to be executed.
+
+${chalk.italic('Script input, only 1 must be provided:')}
+   --stdin   Use the terminal standard input as the script to be executed
+    --file   Path to a file. The content will be sent as the script to be executed
 
 ${chalk.bold('Examples')}
  
-  Execute the content of a SQL script file to the database URL from schema.
+  Execute the content of a SQL script file to the datasource URL taken from the schema
   ${chalk.dim('$')} prisma db execute --preview-feature --file ./script.sql --schema schema.prisma
 
-  Execute the SQL script from stdin to the database url.
+  Execute the SQL script from stdin to the datasource URL specified via the \`DATABASE_URL\` environment variable
```

Comment:
Hm, is this hardcoded to `DATABASE_URL` or is it whatever is defined in `schema.prisma`? Could be harded in there, or be called `FOO_BOOP` instead.